src/App.js

```diff
@@ -1,7 +1,7 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
 import {Navbar,Footer} from "./components/main-component";
-import {Login,Products,Signup,Wishlist,Cart} from "./pages/main";
+import {Login,Products,Signup,Wishlist,Cart,Home} from "./pages/main";
```

Comment:
If you rename `main.js` as `index.js` then you can import the components just by mentioning the folder name. i.e `from "./pages"`